modules/top-level/nixpkgs.nix

```
@@ -56,6 +56,29 @@ in
       '';
     };
 
+    config = lib.mkOption {
+      default = { };
+      example = {
+        allowBroken = true;
+        allowUnfree = true;
+      };
+      type = lib.mkOptionType {
+        name = "nixpkgs-config";
+        description = "nixpkgs config";
+        check = x: if isConfig x then true else lib.traceSeqN 1 x false;
```

```suggestion
        check = x: isConfig x || lib.traceSeqN 1 x false;
```

Is this better or worse?

---

Either is fine to me. Or condition maybe cleaner?